client/views/helpers/chunks/question_div/question_div.scss

```diff
@@ -390,3 +390,24 @@ $popular-question-color: #f1c40f;
   font-weight: 600;
   color: #777!important;
 }
+
+.remote-highlight:hover, .remote-highlight > input:hover {
```

Comment:
@singhsanket143 isn't `.remote-highlight:hover` enough here?

Reply:
@peter-hank Updated this css styling.